{{ cookiecutter.__project_name_kebab }}/.github/workflows/nightly.yml

```diff
@@ -0,0 +1,30 @@
+name: Nightly Wagtail Test
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+    # At 01:00, daily
+  workflow_dispatch:
+
+jobs:
+  nightly-wagtail-test:
```

Comment:
I'm not really convinced there is any value to running the lint checks nightly. That stuff is all pinned. It shouldn't be changing out from under us. I've only migrated the nightly check against wagtail core.

Reply:
I agree. The goal is to check how the package behaves against latest Wagtail core. Linting errors would be caught on `main`